docs/book/src/topics/crd-changes/v1alpha6-to-v1alpha7.md

```diff
@@ -33,3 +33,25 @@ The OpenStackMachine spec previously contained a `subnet` field which could used
 to set the `accessIPv4` field on Nova servers. This feature was not widely
 used, difficult to use, and could not be extended to support IPv6. It is
 removed without replacement.
+
+#### Removal of Port SecurityGroups
+
+The Port field of the OpenStackMachine spec previously contained both `securityGroups` and `securityGroupFilters`.
+As `securityGroups` can easily be replaced with `securityGroupFilters`, that can express the same and more, `securityGroups` has now been removed.
+CAPO can automatically convert `securityGroups` to `securityGroupFilters` when upgrading.
+Downgrading will however *not* give back `securityGroups` but instead retain the `securityGroupFilters`.
+
```

Comment:
I think this is no longer true, right? Should probably also update the PR description.